### PR TITLE
Move all the autoyast profiles from YaST job groups to 'data/yam/'

### PR DESCRIPTION
**Pull request by manfredi**
- Description: Move all the autoyast profiles from _YaST job groups_ to `data/yam/autoyast/`
- Related ticket: https://progress.opensuse.org/issues/131333
- Verifciation runs:
  + [Overview](https://openqa.suse.de/tests/overview?version=15-SP5&distri=sle&build=manfredi%2Fos-autoinst-distri-opensuse%23issues-131333)
  + [ppc64le](https://openqa.suse.de/tests/11616649)
  + [s390x](https://openqa.suse.de/tests/11616650)
-  Related MR [516](https://gitlab.suse.de/qsf-y/qa-sle-functional-y/-/merge_requests/516)

The two test **Failed** of `autoyast_offline_media_install+registration@manfredi/os-autoinst-distri-opensuse#issues-131333` 

-  [ppc64le](https://openqa.suse.de/tests/11617110)
-  [s390x](https://openqa.suse.de/tests/11616717)

were be re-run by adding the var `BUILD="102.1"` as _work around_ to the issue seen in the final _xml_ files due to the fact that they are "*.xml.ep". So the `<media_url>` was wrong.
 - [ppc64le](https://openqa.suse.de/tests/11616649)
 - [s390x](https://openqa.suse.de/tests/11616650)

### CODE REVIEW

**Comment by rakoenig** [comment]
Naming scheme should avoid that `autoyast` is appearing more than one time in the filename.

File: data/yam/sle15_autoyast_create_hdd_gnome.xml
Comment (lemon-suse) [inline data/yam/sle15_autoyast_create_hdd_gnome.xml:1]: This seems very generic, may need check whether other squads use it or not.

File: data/yam/sle15_autoyast_gnome.xml
Comment (lemon-suse) [inline data/yam/sle15_autoyast_gnome.xml:1]: This seems very generic, may need check whether other squads use it or not.

File: data/yam/sle15_autoyast_media_install_gnome.xml
Comment (lemon-suse) [inline data/yam/sle15_autoyast_media_install_gnome.xml:1]: This seems very generic, may need check whether other squads use it or not.

File: data/yam/sle15_autoyast_media_install_gnome_ppc64le.xml.ep
Comment (lemon-suse) [inline data/yam/sle15_autoyast_media_install_gnome_ppc64le.xml.ep:1]: This seems very generic, may need check whether other squads use it or not.

File: data/yam/sle15_autoyast_sles_zvm.xml
Comment (lemon-suse) [inline data/yam/sle15_autoyast_sles_zvm.xml:1]: This seems very generic, may need check whether other squads use it or not.

**Comment by lemon-suse** [comment]
From one of your VR https://openqa.suse.de/tests/11522742#settings, it just use the AUTOYAST: autoyast_sle15/create_hdd/create_hdd_gnome_%ARCH%.xml which used by functional squad, so we should copy it to /data/yam/

**Comment by lemon-suse** [comment]
Please add some PR description such as what the PR will do, the scope, the background, history etc.

Add please follow the PR template to update while not remove items, such as

PR description.

Related ticket: xxx
Needles: xxx
Verification run: xxx

File: data/yam/sle15_autoyast_gnome.xml
Comment (rfan1) [reply data/yam/sle15_autoyast_gnome.xml:1]: The autoyast files used by functional job groups are under `data/autoyast_sle15/create_hdd/`directory. so the changes will  not cause problems.

**Comment by manfredi** [comment]
This PR move all the _autoyast_ profiles from _YaST job groups_ to `data/yam/`.
The list of files, with relative _target_ name are listed below
```
git mv data/autoyast_sle15/autoyast_bcache.xml data/yam/sle15_autoyast_bcache.xml
git mv data/autoyast_sle15/autoyast_btrfs_quota.xml data/yam/sle15_autoyast_btrfs_quota.xml
git mv data/autoyast_sle15/autoyast_btrfs.xml data/yam/sle15_autoyast_btrfs.xml
git mv data/autoyast_sle15/autoyast_create_hdd_gnome.xml data/yam/sle15_autoyast_create_hdd_gnome.xml
git mv data/autoyast_sle15/autoyast_disk_as_md_member.xml data/yam/sle15_autoyast_disk_as_md_member.xml
git mv data/autoyast_sle15/autoyast_disk_as_pv.xml data/yam/sle15_autoyast_disk_as_pv.xml
git mv data/autoyast_sle15/autoyast_error.xml data/yam/sle15_autoyast_error.xml
git mv data/autoyast_sle15/autoyast_eula.xml data/yam/sle15_autoyast_eula.xml
git mv data/autoyast_sle15/autoyast_ext4.xml data/yam/sle15_autoyast_ext4.xml
git mv data/autoyast_sle15/autoyast_firstboot.xml data/yam/sle15_autoyast_firstboot.xml
git mv data/autoyast_sle15/autoyast_gnome.xml data/yam/sle15_autoyast_gnome.xml
git mv data/autoyast_sle15/autoyast_home_encrypted.xml data/yam/sle15_autoyast_home_encrypted.xml
git mv data/autoyast_sle15/autoyast_invalid_default_target.xml data/yam/sle15_autoyast_invalid_default_target.xml
git mv data/autoyast_sle15/autoyast_iscsi_ibft.xml data/yam/sle15_autoyast_iscsi_ibft.xml
git mv data/autoyast_sle15/autoyast_keyboard_layout.xml data/yam/sle15_autoyast_keyboard_layout.xml
git mv data/autoyast_sle15/autoyast_lvm.xml data/yam/sle15_autoyast_lvm.xml
git mv data/autoyast_sle15/autoyast_media_install_gnome_ppc64le.xml.ep data/yam/sle15_autoyast_media_install_gnome_ppc64le.xml.ep
git mv data/autoyast_sle15/autoyast_media_install_gnome_s390x.xml.ep data/yam/sle15_autoyast_media_install_gnome_s390x.xml.ep
git mv data/autoyast_sle15/autoyast_media_install_gnome.xml data/yam/sle15_autoyast_media_install_gnome.xml
git mv data/autoyast_sle15/autoyast_multi_btrfs.xml data/yam/sle15_autoyast_multi_btrfs.xml
git mv data/autoyast_sle15/autoyast_multipath.xml data/yam/sle15_autoyast_multipath.xml
git mv data/autoyast_sle15/autoyast_nfs_share.xml data/yam/sle15_autoyast_nfs_share.xml
git mv data/autoyast_sle15/autoyast_non_existing_graphical_target.xml data/yam/sle15_autoyast_non_existing_graphical_target.xml
git mv data/autoyast_sle15/autoyast_non_secure_boot.xml data/yam/sle15_autoyast_non_secure_boot.xml
git mv data/autoyast_sle15/autoyast_resize_luks2.xml data/yam/sle15_autoyast_resize_luks2.xml
git mv data/autoyast_sle15/autoyast_reuse-encrypted.xml data/yam/sle15_autoyast_reuse-encrypted.xml
git mv data/autoyast_sle15/autoyast_salt.xml data/yam/sle15_autoyast_salt.xml
git mv data/autoyast_sle15/autoyast_sle_powervm.xml.ep data/yam/sle15_autoyast_sle_powervm.xml.ep
git mv data/autoyast_sle15/autoyast_sles_zvm.xml data/yam/sle15_autoyast_sles_zvm.xml
git mv data/autoyast_sle15/autoyast_systemd_timesync.xml data/yam/sle15_autoyast_systemd_timesync.xml
git mv data/autoyast_sle15/autoyast_tftp.xml data/yam/sle15_autoyast_tftp.xml
git mv data/autoyast_sle15/bug-872532_ix64ph1069.xml data/yam/sle15_bug-872532_ix64ph1069.xml
git mv data/autoyast_sle15/bug-876411_btrfs_h5_autoinst.xml data/yam/sle15_bug-876411_btrfs_h5_autoinst.xml
git mv data/autoyast_sle15/bug-877438_ix64ph1029.xml data/yam/sle15_bug-877438_ix64ph1029.xml
git mv data/autoyast_sle15/bug-879147_autoinst.xml data/yam/sle15_bug-879147_autoinst.xml
git mv data/autoyast_sle15/bug-887126_autoinst.xml data/yam/sle15_bug-887126_autoinst.xml
git mv data/autoyast_sle15/bug-887653_autoinst_jy-snapshot.xml data/yam/sle15_bug-887653_autoinst_jy-snapshot.xml
git mv data/autoyast_sle15/mini_remote.xml data/yam/sle15_mini_remote.xml
git mv data/autoyast_sle15/mini.xml data/yam/sle15_mini.xml
git mv data/autoyast_sle15/rule-based_example/ data/yam/sle15_rule-based_example
```

and update the relative `PATH`s inside these files:
```
schedule/yast/autoyast/autoyast_activate_encrypted_volume.yaml
schedule/yast/autoyast/autoyast_ext4.yaml
schedule/yast/autoyast/autoyast_gnome.yaml
schedule/yast/autoyast/autoyast_iscsi_ibft.yaml
schedule/yast/autoyast/autoyast_keyboard_layout.yaml
schedule/yast/autoyast/autoyast_lvm.yaml
schedule/yast/autoyast/autoyast_mini_spvm.yaml
schedule/yast/autoyast/autoyast_rules_and_classes.yaml
schedule/yast/autoyast/autoyast_tftp.yaml
schedule/yast/autoyast_bcache.yaml
schedule/yast/autoyast_btrfs.yaml
schedule/yast/autoyast_disk_as_md_member.yaml
schedule/yast/autoyast_disk_as_pv.yaml
schedule/yast/autoyast_eula.yaml
schedule/yast/autoyast_invalid_default_target.yaml
schedule/yast/autoyast_nfs_share.yaml
schedule/yast/autoyast_non_existing_graphical_target.yaml
schedule/yast/autoyast_non_secure_boot.yaml
schedule/yast/autoyast_resize_luks2.yaml
schedule/yast/autoyast_salt_formulas.yaml
schedule/yast/autoyast_systemd_timesync.yaml
schedule/yast/firstboot/autoyast_y2_firstboot.yaml
```


Related ticket: [issues/131333](https://progress.opensuse.org/issues/131333)
Verification run: [here](https://openqa.suse.de/tests/overview?version=15-SP5&distri=sle&build=manfredi%2Fos-autoinst-distri-opensuse%23issues-131333)

The tests (_autoyast_create_hdd_gnome_):
- [https://openqa.suse.de/tests/11522739](https://openqa.suse.de/tests/11522739)
- [https://openqa.suse.de/tests/11522773](https://openqa.suse.de/tests/11522773)
- [https://openqa.suse.de/tests/11522742](https://openqa.suse.de/tests/11522742)

that have these values for `YAML_SCHEDULE` and `AUTOYAST`:
```
YAML_SCHEDULE: schedule/functional/autoyast_create_hdd/autoyast_create_hdd_gnome.yaml
AUTOYAST: autoyast_sle15/create_hdd/create_hdd_gnome_%ARCH%.xml
```

are all related to `schedule/functional` and not to `schedule/yast`, used by _functional squad_, I guess.
So leave this last unchanged

**Comment by jknphy** [comment]
I'm not sure sle15_ is good name convention, those profiles are for product validation, someone could understand that can be used for any sle15 product in maintenance, which I doubt, because we have been updating each SP, so not garantee.
I would leave profiles not in maintenance without any suffix, so then for maintenance we use specific product or something else, anyway the majority of the profiles are from product, so it is "our" default.

**Review by lemon-suse — APPROVED** [review]

**Comment by jknphy** [comment]
> Naming scheme should avoid that `autoyast` is appearing more than one time in the filename.

are we going to loose the opportunity here to remove `autoyast_*` from the name of the files? I agree with this comment, but not really mandatory.

**Comment by manfredi** [comment]
It has been removed the prefix `autoyast_` in all _autoyast_ xml files

**Comment by rakoenig** [comment]
LGTM.

**Pull request merged**